l10n/de_DE/LC_MESSAGES/remoteevent.po

```diff
@@ -1791,6 +1800,12 @@ msgstr ""
 "Mit dieser Einstellung können sie die Registrierung über die RemoteEvent "
 "Schnittstelle aussetzen, unabhängig von den anderen Einstellungen"
 
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid "If enabled, registration is only possible for logged in users."
+msgstr ""
+"Wenn aktiviert, ist eine Veranstaltungsanmeldung nur für eingeloggte "
```

Comment:
I hate that anglicism, but you're right that *anmelden* appearing in two different meanings in the same sentence is *Mist* :wink: